src/lib.rs

```diff
@@ -1,3 +1,63 @@
+//! This crate provides an iron middleware that handles user login sessions
+//! using an encrypted authentication cookie.
```

Comment:
It's only signed, which means the cookie value is readable by anybody (just not modifiable without rendering the signature invalid).